accessibility-checker-engine/help/IBMA_Color_Contrast_WCAG2AA_PV.mdx

```diff
@@ -16,15 +16,15 @@ import { CodeSnippet, Tag } from "carbon-components-react";
 
 ## Why is this important?
 
-When text and its background colors have less than a 4.5 to 1 contrast ratio it can be difficult for people with moderately low vision to read the text without a contrast-enhancing technology. For larger text of 18 point or more, or bold 14 point text, the text and background colors must give at least a 3 to 1 contrast ratio.
+When text and its background colors have less than a 4.5 to 1 contrast ratio it can be difficult for people with moderately low vision to read the text without a contrast-enhancing technology. For larger text of 18 points or more, or bold 14 points text, the text and background colors must give at least a 3 to 1 contrast ratio.
```

Comment:
The second instance should be '14 point text' not '14 points text'.

Just like we say 'a 13 centimeter length' and 'a length of 13 centimeters', we should say 'text of 18 points' and '14 point text'.